pipelines/build/common/config_regeneration.groovy

```diff
@@ -524,7 +521,7 @@ class Regeneration implements Serializable {
         }
 
         // Make sure the dsl knows if we're building inside the pr tester
-        if (isPRBuilder) {
+        if (jobType == "pr-test") {
```

Comment:
should that be `pr-tester`?  I think I also prefer the isPRBuilder style logic.

Reply:
I've thought about to either keep "boolean" for type, or introduce one string then we can use its value to have different logic. 
main reason i picked up the later one is in current case, it will be one isPRBuilder, isReleaseBuild, isPrototypeBuild and potentially when we have new type, we need to create one more and it is already 19 parameters for the class.

Reply:
yeah, i will update all places from `pr-test` to `pr-tester `including some comments